TaskHouseUnitTests/WorkersControllerUnitTests.cs

```diff
@@ -172,8 +143,8 @@ public async void WorkerController_Update_ReturnsObjectResult_withValidIdAndVali
                 Salt = "upYKQSsrlub5JAID61/6pA=="
             };
 
-            var Result= await controller.Update(Id, worker);
-            var resultAsObject = await controller.Get(Id) as ObjectResult;
+            var Result = controller.Update(worker);
+            var resultAsObject = controller.Get(Id) as ObjectResult;
             var resultObject = resultAsObject.Value as Worker;
 
             Assert.IsType<NoContentResult>(Result);
```

Comment:
The method name refers to it as a test for an ObjectResult, but the assert is for a NoContentResult

Reply:
- just noticed this isn't even your work, goddamn this gigantic pr